src/ucp/wireup/address.c

```diff
@@ -990,7 +998,7 @@ ucs_status_t ucp_address_unpack(ucp_worker_t *worker, const void *buffer,
         return UCS_ERR_UNREACHABLE;
     }
 
-    if (unpack_flags & UCP_ADDRESS_PACK_FLAG_WORKER_UUID) {
+    if (address_header & UCP_ADDRESS_HEADER_FLAG_WORKER_UUID) {
```

Comment:
assert that if user passed UCP_ADDRESS_PACK_FLAG_WORKER_UUID in flags, then UCP_ADDRESS_HEADER_FLAG_WORKER_UUID is present in address_header